reports/proposal.md

```diff
@@ -1,8 +1,55 @@
+# STAA 556: Visual Project Proposal
+
+#### Team Members
+- [Rodney Murray](mailto:Rodney.Murray@colostate.edu)
+- [Brendon Stanley](mailto:Brendon.Stanley@rams.colostate.edu)
+
 ## Introduction
-* Context and background information
-* Study objectives
-* Experiment design
-* Other details
+
+<p style="text-indent: 40px;">
+    Hurricane Ian swept through the Hurricane Belt in 2022 leaving behind 149 deaths in Florida
+and an estimated $113 Billion USD in damages (Florida Department of Law Enforcement; National Centers for Environmental Information).
+Concerns were raised regarding the readability of the hurricane forecast maps and the "cone" used to predict the path of the hurricane (Dance and Ajasa).
+News outlets have suggested that the cone is misleading and that the public may not understand the information presented (Pulver and Doyle).
+This project aims to provide background to better understand how vision works and how to produce informative data visualizations.
+The project will be a stepping stone for continued research of data visualization and how to communicate hurricane forecast data effectively.
+</p>
+
+<p style="text-indent: 40px;">
+    Data for this project was collected in an experiment of 52 subjects enrolled in psychology 100 at CSU. Subjects were shown an image of dots via an online survey (see figure one) and 
+asked to select the quadrant where the centroid of the dots was located (see figure two); i.e., where is the middle of the dots? Each image was show to the subject for less than a second before the
+screen would change. Asking the subject to select the quadrant the centroid of the dots was in. The subjects' response was recorded along with metadata about the image. Each subject was 
```

Comment:
Maybe reword to:

After the image was shown for less than one second, the subject was asked to select which quadrant the centroid of the dots was in.